docs/INSTALL.UBUNTU.md

```diff
@@ -0,0 +1,61 @@
+Installing OVS and OVN from sources on Ubuntu
+=============================================
+
+Install a few pre-requisite packages.
+
+```
+apt-get update
+apt-get install -y build-essential fakeroot debhelper \
+                    autoconf automake bzip2 libssl-dev \
+                    openssl graphviz python-all procps \
+                    python-dev python-setuptools \
+                    python-twisted-conch libtool git dh-autoreconf \
+                    linux-headers-$(uname -r)
+easy_install -U pip
+```
+
+Clone the OVS repo.
+
+```
+git clone https://github.com/openvswitch/ovs.git
+cd ovs
+```
+
+Configure and compile the sources
+
+```
+./boot.sh
+./configure --prefix=/usr --localstatedir=/var  --sysconfdir=/etc --enable-ssl --with-linux=/lib/modules/`uname -r`/build
+make -j3
+```
+
+Install the executables
+
+```
+make install
+make modules_install
```

Comment:
In order to leverage CT-nat you probably don't want to compile the modules from sources but use the kernel ones?